### ISSUE TITLE
Hello, I'm very interested in your work, but the effect of adding drfpn to the network has not been improved. 

### ISSUE THREAD

**Issue by kongyan66** [myAnonRepo/DRFPN#1]
I'm using retinanet as the baseline, replacing FPN with drfpn. In the hrsc2016 dataset, the map is lower than the baseline. Why?
![图片1](https://user-images.githubusercontent.com/68680103/114990492-23235580-9ecb-11eb-8d1c-db0cbd4b53d1.png)

**Comment by myAnonRepo** [comment]
Sorry to reply to you so late. 
As far as I know, the hrsc2016 dataset is a public dataset of remote sensing ships. Most of the objects are densely arranged. This is the difference between it and the general object dataset COCO. Our experiments have found that although DRFPN can effectively correct the bounding box position of general objects, the detection performance for densely arranged objects does not seem to be ideal. We assume that this is because the feature deformation of densely arranged objects weakens the recall of the model. 
Our follow-up work will improve this problem. Finally, thanks for your attention to our work.  :)

**Comment by kongyan66** [comment]
Thank you for your accurate reply. I'm also doing the work of remote sensing target feature enhancement recently. I hope to have the opportunity to communicate more. Lastly, look forward to your better works.

**Comment by dailonggang** [comment]
I used Faster R-CNN as the baseline and replaced the FPN with DRFPN, and the mAP was much lower than the baseline in the VOC2007 dataset, why is that?

**Comment by myAnonRepo** [comment]
> I used Faster R-CNN as the baseline and replaced the FPN with DRFPN, and the mAP was much lower than the baseline in the VOC2007 dataset, why is that?

In fact, after the experiments, we found that the performance on the VOC are indeed not as ideal as the COCO. We assume that the reason for this situation is a different AP calculation method between VOC and COCO. COCO evaluated APs under 10 IoUs ranging from 0.5 to 0.95, and finally took the average of APs under these thresholds as the result. In VOC, the test result only evaluates the AP value of IOU under the threshold of 0.5. The evaluation style of the COCO method not only focuses on the classification ability of the model, but also focuses on the positioning ability of the model.  When visualizing the DRFPN, we found that DRFPN will condense the feature activation areas inside the object, whiche will indeed reduce the detection recall. But the locating performance of the detection is better. Maybe a different mAP calculation formula is the reason.
